### ISSUE TITLE
Use "latest" client instead of "2.2" link in desktop client documentation

### ISSUE THREAD

**Issue by RealRancor** [owncloud/client#5308]
Hi,

i think the same as done for the server documentation in https://github.com/owncloud/documentation/issues/2646 could also make sense for the docs of the desktop client. So instead of:

https://doc.owncloud.org/desktop/2.2/

a link like:

https://doc.owncloud.org/desktop/latest/

could be used. This would prevent e.g. that a search on google for:

https://www.google.de/#q=install+owncloud+client

is currently showing the 1.8 docs as the second hit.

cc @carlaschroder @jnweiger

**Comment by guruz** [comment]
I had those discussions before with @danimo 
Isn't robots.txt etc what nudges google to pick a better link?

**Comment by carlaschroder** [comment]
@jnweiger What do you think? Are there server logs that can tell us if this is working?

**Comment by carlaschroder** [comment]
We have a real problem. I did a number of searches such as 'install owncloud', 'owncloud user management', and other common admin tasks, and looks what comes up. Old stuff always comes first.

![owncloud-search](https://cloud.githubusercontent.com/assets/8388387/20467112/08856cb6-af35-11e6-9710-12cc94bc83f2.png)

**Comment by RealRancor** [comment]
Hi,

> Old stuff always comes first.

yeah. I think it will take quite some  time until the new "latest" docs are getting a higher score on google then 3 year old docs. But it is in the google index and probably will show up on top in a few month

https://www.google.de/#q=doc.owncloud.org+inurl:latest

**Comment by settermjd** [comment]
I agree that using **"latest"** is much more effective — especially over the longer term — than a specific version number. It would simplify a host of links and make links simpler to remember. Plus, you can always point at, so the docs never go out of date.

**Comment by settermjd** [comment]
What I suggest doing, for all of the online documentation, is implementing sitemap.xml files, if they're not already. They're not in place for the main documentation. I'll have to check for other repositories.

**Comment by settermjd** [comment]
Not sure why I didn't think of this earlier, likely because I'm not that familiar with Jenkins. But we can change the remote directory setting to be `desktop/latest` instead of `desktop/2.3` (the latest release). I'm quickly testing that at the moment, and will revert it afterward.

**Comment by SamuAlfageme** [comment]
Redirection alias on the doc's `.htaccess` from `/latest` to `/2.3` would be the way to go to avoid rebuilding stuff...

```
Redirect /desktop/latest https://doc.owncloud.org/desktop/2.3/
```

**Comment by settermjd** [comment]
I don't see how that would work honestly. There'd still need to be a Jenkins build update to point the latest branch to "latest". Am I missing something?

**Comment by SamuAlfageme** [comment]
@settermjd yeah, but having it as alias would mean that both`desktop/2.3` and `/desktop/latest` would work. **Is important to have both available as the link to the docs in the client (tray menu icon > Help) relies on the version to point to the right docs.** 

I meant "rebuilding" as in building `2.3` docs once `2.4` is live because up until then it was built with the `/latest` path. Sorry if I was not clear on my first comment 😅
